sqlx-macros/src/database/mysql.rs

```diff
@@ -46,6 +46,9 @@ impl_database_ext! {
 
         #[cfg(feature = "bigdecimal")]
         sqlx::types::BigDecimal,
+
+        #[cfg(feature = "decimal")]
+        sqlx::types::Decimal,
```

Comment:
Something to note is that when multiple types are compatible with the same SQL type, like `BigDecimal` and `Decimal` are both compatible with `NUMERIC`, the first one declared (and enabled) is the one the macros will pick for typechecking.

It's the same situation with types from `chrono` and `time`, although I don't think we document that either so I won't block this PR on it.